cardea/problem_definition/definition.py

```diff
@@ -62,6 +84,57 @@ def generate_cutoff_times(self, entity_set):
             ValueError: An error occurs if the cutoff variable does not exist.
         """
 
+        target_label_exists = DataLoader().check_column_existence(
```

Comment:
Instead of creating a new `DataLoader()` object 3 times, can we create a `data_loader` variable and invoke its methods each time? Not entirely sure, but I believe this is safe because we include the parameters within the method.

Reply:
definitely, I agree. Changes --> 609d4c0b94b422f5b6f342cf55fd1b39fec01fd5